test/Microsoft.Framework.ConfigurationModel.Test.Common/IConfigurationSourceExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.ConfigurationModel
+{
+    public static class IConfigurationSourceExtensions
```

Comment:
1. Do we need an extension method for this? If so, why is it in a test? Wouldn't this benefit everyone? Or is it really needed at all?
2. If we keep it, the naming pattern for extension methods is here: https://github.com/aspnet/Home/wiki/Engineering-guidelines#extension-method-patterns